backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/bottomsheet/BpkBottomSheetState.kt

```diff
@@ -18,78 +18,73 @@
 
 package net.skyscanner.backpack.compose.bottomsheet
 
-import androidx.compose.animation.core.AnimationSpec
-import androidx.compose.material.ExperimentalMaterialApi
-import androidx.compose.material.SwipeableDefaults
-import androidx.compose.material.SwipeableState
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.SheetState
+import androidx.compose.material3.SheetValue
+import androidx.compose.material3.rememberStandardBottomSheetState
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.Stable
-import androidx.compose.runtime.saveable.Saver
-import androidx.compose.runtime.saveable.rememberSaveable
-import net.skyscanner.backpack.compose.BpkSwipeableState
+import androidx.compose.runtime.remember
 
 @Composable
+@OptIn(ExperimentalMaterial3Api::class)
 fun rememberBpkBottomSheetState(
-    initialValue: BpkBottomSheetValue,
-    animationSpec: AnimationSpec<Float> = SwipeableDefaults.AnimationSpec,
```

Comment:
what happened to this?

Reply:
AnimationSpec is not a passable parameter to MD3 bottom sheet state anymore.